ts/Core/Chart/Chart.ts

```diff
@@ -3762,10 +3764,14 @@ class Chart {
                     }
 
                     // set new reference for next run:
-                    (chart as any)[mouseDown] = mousePos;
+                    nextMousePos[mouseDown] = mousePos;
                 }
             });
 
+            objectEach(nextMousePos, (pos, down): void => {
+                (chart as any)[down] = pos;
+            });
```

Comment:
It would be nice if we take this opportunity to refactor this. We already have `Pointer.mouseDownX` and `Pointer.mouseDownY`. Can you see if we can work on them instead of having additional and similar props on the `Chart`?

Reply:
The `Chart` props are updated on every `pan` call since `pan` needs to know how far the cursor has moved since the previous call, while the `Pointer` props is the actual mousedown position. So it seems we would need some sort of state in addition to the `Pointer` props anyway.

Reply:
Yes... Actually the `Chart` props have a misleading name. It is not the _mouse down X_, it is actually the X of the last pan event. Okay, let's just leave it for now since this PR is about something else.